pkg/arvo/sys/vane/ames.hoon

```diff
@@ -288,8 +288,7 @@
   ^-  [sig=@ux dat=$@(~ (cask))]
   =/  mes=@
     %+  rep  response-size
-    %+  turn
-      (sort hav |=([a=have b=have] (lth fra.a fra.b)))
+    %+  turn  (flop hav)
```

Comment:
Switching this back to `(roll hav |=([=have dat=(list @ux)] [dat.have dat]))` reduces packet assembly traversals from 3 to 2.